README.md

```diff
@@ -373,15 +373,22 @@ will give a more in-depth description of the command and available options.
 #### Proxy Submissions
 
 The autograder allows course staff to submit on behalf of students using "proxy" submissions.
-The proxy submit command adds two parameters to the normal submission command:
+The proxy submit commands add two parameters to the normal submission command:
 
  - `proxy-email` -- The student that the course staff is submitting on behalf of.
  - `proxy-time` -- The "fudged" submission time that is automatically set to the earlier time between now and one minute before the due date.
```

Comment:
In the rest of this document we prefix flags (in inline code) with a `--`.
So, `proxy-email` -> `--proxy-email`.